tests/monitoring/component_monitoring.go

```diff
@@ -191,6 +195,13 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 			if !errors.IsAlreadyExists(err) {
 				util.PanicOnError(err)
 			}
+			operatorRoleBinding.Annotations = nil
+			operatorRoleBinding.ObjectMeta.ResourceVersion = ""
+			operatorRoleBinding.ObjectMeta.UID = ""
+			_, err = virtClient.RbacV1().RoleBindings(flags.KubeVirtInstallNamespace).Create(context.Background(), operatorRoleBinding, metav1.CreateOptions{})
+			if !errors.IsAlreadyExists(err) {
+				Expect(err).ToNot(HaveOccurred())
+			}
```

Comment:
```suggestion
			Expect(err).To(Or(MatchError(errors.IsAlreadyExists, "IsAlreadyExists"), Not(HaveOccurred())))
```
I would align the code also in lines 195-197

Reply:
My bad:
```
Expect(err).To(Or(Not(HaveOccurred(), MatchError(errors.IsAlreadyExists, "IsAlreadyExists"))))
```

Reply:
@RamLavi I think you have to change the Expect as I suggested before retest

Reply:
oh I missed that. you're right. will change.

Reply:
Changed to:
```
Expect(err).To(Or(Not(HaveOccurred()), MatchError(errors.IsAlreadyExists, "IsAlreadyExists")))
```